Training/2.0 Libraries.md

```diff
@@ -25,14 +26,22 @@ Make a two-dimensional numpy array. First row will be: [1,2,3] and second row wi
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
 ```
-
+import numpy as np
+array = np.array([[1,2,3],[4,5,6]])
+mean = array.mean()
+max = array.max()
+min = array.min()
 ```
 
 ### Task 2b)
 Calculate the trimmed mean of the following array using ONLY Numpy and python's base functions: [67,67,65,50,69,68,67,70,69,66]. The trimmed mean is the mean of the data with the largest and the smallest values (outliers) removed.
 
 ```
-
+import numpy as np
+from scipy import stats
+x = 67,67,65,50,69,68,67,70,69,66
+a = stats.trim_mean(x, 0.1)
+print(a)
```

Comment:
Finding a library function for the job is a good approach; you should certainly do this when you encounter a new coding problem, because chances are someone else may already have written a function for it!

But for this task, the prompt says use **only** Numpy and python's base functions, so please try to solve it without scipy.